doc/release-notes/8424-signposting.md

```diff
@@ -0,0 +1,42 @@
+# Signposting for DV 5.9
+
+This branch adds [Signposting](https://signposting.org/) support to DV 5.9.
+
+There are 2 Signposting profile levels, level 1 and level 2. In this implementation, 
+ * level 1 links are shown in 
+HTTP header, which can be fetched by `curl -I https://domain/link-to-article`. 
+ * The level 2 linkset can be fetched by visiting the dedicated linkset page for 
+   that artifact. The link can be seen in level 1 links with key name `rel="linkset"`.
+
+The configuration is stored as JSON string in the `Bundle.properties` file, key name is
+`signposting.configuration.SignpostingConf`. Please see a sample configuration below with explaination for each of the
+config items.
+
+```json
+{
+  "license": {
+    "CC0": "https://creativecommons.org/licenses/cc0/"
+  },
+  "describedby": {
+    "doi": "https://doi.org/",
+    "type": "application/vnd.citationstyles.csl+json"
+  },
+  "useDefaultFileType": true,
+  "defaultFileTypeValue": "https://schema.org/Dataset",
+  "maxItems": 5,
+  "maxAuthors": 5
+}
```

Comment:
The signposting cannot be disabled :D 

The parameters can be set to the preference of the users. We wanted to make it slightly customizable as in the first draft these are not changeable and the level 1 header becomes really messy and difficult to read, even though it is not particularly set for human to read. ;) 

For example, for now Herbert is ok with a default file type set to `https://schema.org/Dataset`. This might change when there is an upstream schema change or he changes his idea. 
`maxItems` determines max items/files to show in Level 1 profile, as it can get really lengthy. Same goes for `maxAuthor`.